README.md

```diff
@@ -43,7 +44,7 @@ Building the code
    ```
    $ mkdir build && cd build
    $ cmake -DCMAKE_INSTALL_PREFIX=/opt/dirtsand ..
-   $ make && sudo make install
+   $ cmake --build . && sudo cmake --install .
```

Comment:
`cmake --install` was added in 3.15, which conflicts with the version requirement added above...  Might be better to say `sudo cmake --build . --target install`, which also works with older versions.